src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -36,6 +36,86 @@ import './MetadataSidebarRedesign.scss';
 import MetadataInstanceEditor from './MetadataInstanceEditor';
 import { convertTemplateToTemplateInstance } from './utils/convertTemplateToTemplateInstance';
 
+const SUPPORTED_FILE_EXTENSIONS = new Set([
```

Comment:
I think we should move this `const` to its own file since it taking up half of the file